Multisig/UI/Safe Management/Safe Settings/SafeSettingsContentView.swift

```diff
@@ -36,13 +38,13 @@ struct SafeSettingsContentView: View {
             }
 
             Section(header: ListSectionHeader(text: "ENS NAME")) {
-                LoadableENSNameText(safe: safe, placeholder: "Not Set")
+                LoadableENSNameText(safe: safe, placeholder: "Not set").frame(height: rowHeight)
             }
 
             Section(header: ListSectionHeader(text: " ")) {
                 NavigationLink(destination: SafeAdvancedSettingsView(safe: safe)) {
                     BodyText("Advanced")
-                }
+                }.frame(height: rowHeight)
```

Comment:
what is our style convention? are we putting .frame next to "{" or on the next line like below with padding?

Reply:
As it's only one attribute I thought to put it on same line, but now i make them separated